### PR TITLE
[Tests-Only] Splitted acceptance-test-suite webUISharingInternalUsers

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Related Issue

- Part of #36859

## Motivation and Context
- See #36859 

## How Has This Been Tested?

- :robot:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: .drone.starlark [hunk 1]
```diff
@@ -161,7 +161,8 @@ config = {
 				'webUISharingAcceptShares': 'webUISharingAcceptSh',
 				'webUISharingAutocompletion': 'webUISharingAutocomp',
 				'webUISharingInternalGroups': 'webUISharingIntGroup',
-				'webUISharingInternalUsers': 'webUISharingIntUsers',
+				'webUISharingInternalUsers1': 'webUISharingIntUsers1',
+                'webUISharingInternalUsers2': 'webUISharingIntUsers2',
```
Comment (phil-davis) [inline .drone.starlark]: indent

**Review by phil-davis — COMMENTED** [review]
We should also look through the `shareWithUsers` feature and see if we can reorganise it into a few different features with different names. But we can do that in a follow-up PR.

**Comment by phil-davis** [comment]
@kiranparajuli589 now we started to get conflicts. Please rebase and sort out the indent.

File: .drone.starlark (hunk 1 above)
Comment (kiranparajuli589) [reply .drone.starlark]: :+1:

**Comment by kiranparajuli589** [comment]
> We should also look through the `shareWithUsers` feature and see if we can reorganise it into a few different features with different names. But we can do that in a follow-up PR.

Fine, I'd make an issue and assign myself on it!

**Review by phil-davis — APPROVED** [review]

**Pull request merged**